src/env-var.ts

```diff
@@ -0,0 +1,11 @@
+const getSkipDestroyEnvVar = (): boolean => {
+  // if we are in a node environment, check the process.env object
+  if (typeof process !== 'undefined' && 'env' in process) {
+    return (process.env.TFF_SKIP_DESTROY ?? '').trim().length > 0
```

Comment:
Maybe do a truthy check here? `TFF_SKIP_DESTROY=0` (as opposed to the documented `TFF_SKIP_DESTROY=1`) will still trigger this behaviour, right?

Reply:
That's a good point, I'll do that

Reply:
Env var parsing is weird, I decided to embed the `yn` library which handles yes/no, y/n, 1/0, etc